### PR TITLE
Updated README.md to reflect version 3.0.0

### PR DESCRIPTION
**Pull request by jarretlavallee**
Fixes issue #8

Prior to this commit the *README.md* showed information about v2.x, which
is no longer accurate with v3.0.0. This update provides new output and
updated commands for the current state of the module in the *README.md*. 

Since there is a dependency on *puppetlabs-stdlib*, I added `".:$(puppet config print modulepath)"` as the module path for the puppet apply. This should hopefully exclude users from having to download the *puppetlabs-stdlib* module locally to */tmp*. I figure that most customers will have the *stdlib* module already installed, but the *README.md* can easily changed to show how to pull the *puppetlabs-stdlib* module into */tmp* instead if that is a better approach.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,154 +1,187 @@
...
-    * [Split Install ( Running on the Master )](#split-install--running-on-the-master-)
-    * [Monolithic With Compile Masters ( Running on the MoM )](#monolithic-with-compile-masters--running-on-the-mom-)
-    * [Split With Compile Masters ( Running on the MoM )](#split-with-compile-masters--running-on-the-mom-)
-    * [Running on PE 3\.8](#running-on-pe-38)
-    * [Other Option](#other-option)
-  * [What do you get](#what-do-you-get)
-    * [Grepping for Metrics](#grepping-for-metrics)
-      * [Puppetserver](#puppetserver)
-      * [PuppetDB](#puppetdb)
+- [How to use](#how-to-use)
```
Comment (npwalker) [inline README.md]: not obvious but I'm generating this from gh-md-toc.  https://github.com/ekalinin/github-markdown-toc

**Comment by jarretlavallee** [comment]
TOC has been updated with * instead of - for the bullet points and better nesting.

File: README.md [hunk 2]
```diff
@@ -3,152 +3,237 @@ Table of Contents
...
+ - 'master-1.domain.com'
+ - 'compile-master-1.domain.com'
+ - 'compile-master-2.domain.com'
+~~~
+
+### Class Definition Example
+
+~~~
+class { 'pe_metric_curl_cron_jobs':
+  puppet_server_hosts => [
```
Comment (reidmv) [inline README.md]: this should be `puppetserver` (current API) instead of `puppet_server` (deprecated API)

File: README.md [hunk 3]
```diff
@@ -3,152 +3,237 @@ Table of Contents
...
 
-Refer to the other examples if you want to change other parameters.
+This option puts metrics on each individual node instead of gathering metrics centrally on the CA master.  In order to do so, you would classify each of your PE infrastructure nodes with this module.  This option is discouraged but if
+for some reason you can't reach out across a network segment or some other reason you may still wish to have metrics.
 
-## Other Option
+When you classify a compile master you would set `$puppetdb_metrics_ensure` to `absent`.
 
-This option puts metrics on each individual node so I don't think it's as good as having centrally gathered metrics but you can also install the module on each individual node.  If you install on a compile master you can set `puppetdb_metrics_ensure` to `absent` and if you install on a puppetdb node then you can set `$puppet_server_metrics_ensure` to `absent`.
+When you classify a PuppetDB node you would set `$puppet_server_metrics_ensure` to `absent`.
```
Comment (reidmv) [inline README.md]: ~`puppet_server`~ => `puppetserver`

**Review by reidmv — CHANGES_REQUESTED** [review]
This is great! I love it when people update the docs. :)

Dropped a few comments about missing ~`puppet_server`~ => `puppetserver` conversions. Otherwise, all 👍 !

**Pull request merged**